cmd/kubeadm/app/cmd/upgrade/common.go

```diff
@@ -70,16 +70,19 @@ func enforceRequirements(flags *cmdUpgradeFlags, dryRun bool, newK8sVersion stri
 		cfg.KubernetesVersion = newK8sVersion
 	}
 
+	// If features gates are passed to the command line, use it (otherwise use featureGates from configuration)
```

Comment:
It'd be good to tell users this in the flag's description. They should know the behavior, IMO.

Reply:
Agreed, but I'm not going to block on this and with the overhaul coming to CLI+phases+Config we will add that documentation. 

/cc @liztio @fabriziopandini